test/API/H5_api_file_test.c

```diff
@@ -1950,122 +1940,6 @@ test_file_open_overlap(void)
                  "connector\n");
         return 0;
     }
-
-#ifndef NO_DOUBLE_OBJECT_OPENS
```

Comment:
Looks like the whole test below was accidentally deleted

Reply:
I was wondering where that test went. I can move the test back.